eventfinder/src/eventcard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
It's usually best practice to capitalize react component files.

Reply:
I tried changing 'react' to 'React' and received this error.

Cannot find file: 'index.js' does not match the corresponding name on disk: './node_modules/React/react'.

I found this on stackOverflow, and changed it back.

> I come across the same problem.
> 
> I replaced :
> import React, {Component} from 'React', with :
> import React, {Component} from 'react'.
> 
> React is case-sensitive, so be careful and good luck.

Reply:
I think my explanation was poor because I am not able to comment on the name of the file, so I had to choose the first line. What I meant was that the filename `eventcard.js` should be capitalized, although it does not seem to be relevant anymore since it looks to have been deleted.